lib/liquid/ruby_backports.rb

```diff
@@ -0,0 +1,13 @@
+module Liquid
+  module RubyBackports
+    Liquid.private_constant :RubyBackports
+
+    if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.4')
```

Comment:
Benchmarks show that `RbConfig::CONFIG.values_at('MAJOR', 'MINOR').join.to_i < 24` is faster..
```ruby
Benchmark.ips do |x|
  x.report("rbconfig") { RbConfig::CONFIG.values_at('MAJOR', 'MINOR').join.to_i < 24 }
  x.report("gemver") { Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.4') }
  x.compare!
end
```

Reply:
Why does this matter? It's run once at load time.

Reply:
yes, but since Refinements need to be repeated on every *file* for the constituent code to be *refined*, I figured having multiple instances of `Gem::Version` for no other particular reason to be counter-productive..

Reply:
This is happening outside of the `refine` block so it should only get executed once. This also means the refinement would be empty and have no effect on ruby 2.4+.